### DISCUSSION TITLE
Do you know if the plugins are compatible with architect s390x?

### DISCUSSION THREAD

**Discussion by Darkcheat** [grafana/grafana-infinity-datasource#592] [Q&A]
The plugins looks great and would meet my expectations but I don't know if it is compatible with an s390x architecture?

**Comment by yesoreyeram** [comment]
No it doesn't AFAIK. Are you familiar with any other plugins that support?

**Reply by Darkcheat** [reply]
Thank you for your answer, too bad it would have been very useful. I wanted to install this plugins to visualize internal data for my company on an s390x architecture.

**Comment by mykael22000** [comment]
Not support, but Grafana OSS and the included plug-ins have been ported to s390. Not supported by Grafana though.
